controllers/beer.ctrl.js

```diff
@@ -1,10 +1,19 @@
+const { response } = require("../app");
+const Bar = require("../database/models/bar.model");
+const Beer = require("../database/models/beer.model");
+const Beer = require("../models/beer.model");
+
 module.exports.getBeers = (req, res) => {
   res.status(200).json({ message: "Liste de toutes les beers" });
 };
 
-module.exports.getBiere = (req, res) => {
-  const beer_id = req.params.beer_id;
-  res.status(200).json({ message: `Détails de la bière avec l'ID ${beer_id}` });
+module.exports.getBeersByBar = (req, res) => {
+  res.status(200).json({});
+};
+
+module.exports.getBeer = (req, res) => {
+  const id_beer = req.params.id_beer;
```

Comment:
Il faut changer le nom de la variable id_beer => beer_id
```suggestion
  const beer_id = req.params.beer_id;
```